101-redis-cache/azuredeploy.json

```diff
@@ -92,9 +92,10 @@
       },
       "resources": [
         {
-          "type": "Microsoft.Cache/redis/providers/diagnosticsettings",
+          "scope": "[format('Microsoft.Cache/redis/{0}', parameters('redisCacheName'))]",
```

Comment:
doesn't look like we can use scope on a nested child resource...

Reply:
@bmoore-msft - should I just refactor this so that it isn't a nested child? It's not currently taking advantage of the shortened nested child type syntax.

Reply:
yeah I think we should just promote it so we can use the scope property for these samples - aside, seems like we should fix it in the template engine?

Reply:
Possibly? To me that implies that you can nest a resource which isn't actually a child using this syntax. I imagine the only scenario we would actually want to support is scope == the parent resource id (e.g. you're deploying an extension resource)? Worth a discussion!

Reply:
or we just get rid of nesting ;)